### DISCUSSION TITLE
Discussion - the devhub should have a place for users to share examples

### DISCUSSION THREAD

**Discussion by jeremyyeo** [dbt-labs/docs.getdbt.com#1268] [Archive]
The devhub should have a place for examples / patterns / playbooks. These artefacts are shorter than typical dev blog posts (possibly less theoretical and more practical) and its purpose is to answer a **very specific** "how to do X with dbt" question (and likely combines multiple dbt concepts into one).

Examples (and the dbt concepts they use):
* https://github.com/dbt-labs/docs.getdbt.com/discussions/1255 - dbt Cloud API, macros
* https://github.com/dbt-labs/docs.getdbt.com/discussions/1254 - macros, run_query, hooks, variables
* https://gist.github.com/jeremyyeo/5e3bae4e7a2ff6c6b554880c0d8d0e86 - macros, hooks, graph, exceptions
* https://gist.github.com/jeremyyeo/57d09c7e1d4fe31e265a002d30078e3a - dbt Cloud job run status understanding, macros, run_query
* https://gist.github.com/jeremyyeo/7da6a6a4fd6dba598c04c431f74e91c0 - snapshots, customising snapshot source code

Left another comment https://github.com/dbt-labs/docs.getdbt.com/discussions/1255#discussioncomment-2417822 about this topic too:

> Most of the time, these are spurred by customers / community users asking specifically and in this case "do you have a full example of overriding the database name in a dbt Cloud job when we trigger a job via the API". We could certainly share the entire contents above in the body of an email (but certainly that would be less ideal since we'd have to copy paste the content many times).

> These are really not discussions per se - more like patterns/examples on how to do XYZ with dbt and are very specific and at the same time not specific enough (Schrödinger's specificity 😁). Trying to establish a section in the docs site / dev hub for such content that are not long form articles but yet perhaps just too long of an instruction set to just write out in a Slack message / email.

**Comment by jasnonaz** [comment]
Jeremy I completely agree - having a space like these is so important. These writeups are awesome and really appreciate you putting them together.

I _think_ the place for them is actually here, in Discussions, with a dedicated section. I wonder if we want to call it **how-to** or something like that. I wonder what @gwenwindflower thinks.

**Comment by gwenwindflower** [comment]
This was a great suggestion and we did it -- creating this new category type https://github.com/dbt-labs/docs.getdbt.com/discussions/1282
